.github/workflows/connect.yml

```diff
@@ -0,0 +1,31 @@
+# Workflow to connect to a runner
+name: connect
+
+# Trigger the workflow manually
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: build                        # Name of the job
+    runs-on: ubuntu-latest             # Operating system for the job
+
+    steps:
+      - name: Checkout                 # Step to checkout the repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}       # Use the current Git reference
+
+      - name: Set up Python            # Step to set up Python
+        uses: actions/setup-python@v4  # Use v4 for compatibility with pyproject.toml
+        with:
+          python-version: '3.10'       # Specify the Python version
```

Comment:
it's totally possible that we have an issue with only a single version of python, so probably want to have the same build matrix as `ci.yml`

Reply:
I'll set the ci to run on multiple python versions